readme.md

```diff
@@ -5,22 +5,41 @@
 [![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
 [![Join the chat at https://gitter.im/LearningLocker/learninglocker](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/LearningLocker/learninglocker?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-*Learning Locker is a trademark of [HT2 Inc.](http://ht2labs.com)*
+*Learning Locker is a trademark of [Learning Pool](http://learningpool.com)*
 
-### Installation
+### Development: Installation
+
+1. [Install the latest version of Git for version control.](https://git-scm.com/downloads)
```

Comment:
@ryansmith94 most of the list items are labeled `1.`

Reply:
Replied on Slack.